models/user_repository.go

```diff
@@ -0,0 +1,37 @@
+package models
```

Comment:
So, if the repository is not going to return objects conforming to JSON API structure. I can convert it from the main package, right?